crates/katana/storage/provider/src/providers/fork/backend.rs

```diff
@@ -704,4 +878,61 @@ mod tests {
         assert_eq!(class_hash, class_hash_in_cache, "value must be stored in cache");
         assert_eq!(storage_value, storage_value_in_cache, "value must be stored in cache");
     }
+
+    #[test]
+    fn fetch_non_state_data_from_fork() {
+        let (backend, _) = create_forked_backend(LOCAL_RPC_URL.into(), 1);
+
+        assert!(
+            backend
+                .do_get_events(
+                    EventFilter {
+                        from_block: Some(starknet::core::types::BlockId::Tag(BlockTag::Latest)),
+                        to_block: Some(starknet::core::types::BlockId::Tag(BlockTag::Latest)),
```

Comment:
Isn't pending that you want to use here?

Reply:
@glihm Thank you for your comments. Could you share more details or relevant documentation?

Reply:
Currently you're reading `BlockTag::Latest`, and I was wondering if instead we want to use `BlockTag::Pending`.
But re-reading your test, I think it's better what you've did with the latest. @kariy could you confirm?